src/xml_utils.py

```diff
@@ -134,85 +135,79 @@ def format_experiment_xml(library):
     try:
-        xml_exp_set = ET.Element("EXPERIMENT_SET")
-        xml_exp = ET.SubElement(xml_exp_set, "EXPERIMENT")
-        xml_e_identifiers = ET.SubElement(xml_exp, "IDENTIFIERS")
-        xml_e_submitter_id = ET.SubElement(xml_e_identifiers, "SUBMITTER_ID")
+        xml_exp_set = etree.Element("EXPERIMENT_SET")
+        xml_exp = etree.SubElement(xml_exp_set, "EXPERIMENT")
+        xml_e_identifiers = etree.SubElement(xml_exp, "IDENTIFIERS")
+        xml_e_submitter_id = etree.SubElement(xml_e_identifiers, "SUBMITTER_ID")
         xml_e_submitter_id.set("namespace", library['center'])
-        xml_e_submitter_id.text = library["sample_ID"]
-        xml_title = ET.SubElement(xml_exp, "TITLE")
+        xml_e_submitter_id.text = library["sample_id"]
+        xml_title = etree.SubElement(xml_exp, "TITLE")
         xml_title.text = library["title"]
-        xml_study_ref = ET.SubElement(xml_exp, "STUDY_REF")
+        xml_study_ref = etree.SubElement(xml_exp, "STUDY_REF")
         xml_study_ref.set("accession", library["phs_accession"])
-        xml_design = ET.SubElement(xml_exp, "DESIGN")
-        xml_des_desc = ET.SubElement(xml_design, "DESIGN_DESCRIPTION")
+        xml_design = etree.SubElement(xml_exp, "DESIGN")
+        xml_des_desc = etree.SubElement(xml_design, "DESIGN_DESCRIPTION")
         xml_des_desc.text = library["design_description"]
-        xml_smp_desc = ET.SubElement(xml_design, "SAMPLE_DESCRIPTOR")
-        xml_smp_desc.set("refname", library["sample_ID"])
+        xml_smp_desc = etree.SubElement(xml_design, "SAMPLE_DESCRIPTOR")
+        xml_smp_desc.set("refname", library["sample_id"])
         xml_smp_desc.set("refcenter", library["phs_accession"])
-        xml_lib_desc = ET.SubElement(xml_design, "LIBRARY_DESCRIPTOR")
-        xml_lib_name = ET.SubElement(xml_lib_desc, "LIBRARY_NAME")
-        xml_lib_name.text = library["sample_ID"]
-        xml_lib_strat = ET.SubElement(xml_lib_desc, "LIBRARY_STRATEGY")
+        xml_lib_desc = etree.SubElement(xml_design, "LIBRARY_DESCRIPTOR")
+        xml_lib_name = etree.SubElement(xml_lib_desc, "LIBRARY_NAME")
+        xml_lib_name.text = library["sample_id"]
+        xml_lib_strat = etree.SubElement(xml_lib_desc, "LIBRARY_STRATEGY")
         xml_lib_strat.text = library["strategy"]
-        xml_lib_src = ET.SubElement(xml_lib_desc, "LIBRARY_SOURCE")
+        xml_lib_src = etree.SubElement(xml_lib_desc, "LIBRARY_SOURCE")
         xml_lib_src.text = library["source"]
-        xml_lib_sel = ET.SubElement(xml_lib_desc, "LIBRARY_SELECTION")
+        xml_lib_sel = etree.SubElement(xml_lib_desc, "LIBRARY_SELECTION")
         xml_lib_sel.text = library["selection"]
-        xml_lib_layout = ET.SubElement(xml_lib_desc, "LIBRARY_LAYOUT")
-
-        if library["library_layout"] == 'paired':
-            ET.SubElement(xml_lib_layout, "PAIRED")
-        elif library["library_layout"] == 'single':
-            ET.SubElement(xml_lib_layout, "SINGLE")
-        else:
-            sys.exit(4)
```

Comment:
This value was hardcoded as `paired` in library creation so why this is here I'll never know